config/api.yml

```diff
@@ -352,7 +352,10 @@
     :collection_actions:
       :get:
       - :name: read
-        :identifier: ops_settings
+        :identifier:
+        - ops_settings
+        - sui_services_show
```

Comment:
@jvlcek, @chalettu Looking at this, it's not clear why a user should need to have `sui_services_show` and `sui_vm_details_view` to be able to request the `Category` collection. This change would no longer a user to request this collection when he previously could and the reason does not seems obvious.